runtime/engine/src/test/java/io/aklivity/zilla/runtime/engine/test/EngineRule.java

```diff
@@ -218,15 +218,21 @@ public Statement apply(
             Configuration config = description.getAnnotation(Configuration.class);
             if (config != null)
             {
+                String resourceName;
                 if (configurationRoot != null)
                 {
-                    String resourceName = String.format("%s/%s", configurationRoot, config.value());
+                    resourceName = String.format("%s/%s", configurationRoot, config.value());
 
                     configURL = testClass.getClassLoader().getResource(resourceName);
+                    if (configURL == null)
+                    {
+                        configURL = new URL(testClass.getClassLoader().getResource(configurationRoot).toString() +
+                            "/" + config.value());
+                    }
                 }
                 else
                 {
-                    String resourceName = String.format("%s-%s", testClass.getSimpleName(), config.value());
+                    resourceName = String.format("%s-%s", testClass.getSimpleName(), config.value());
```

Comment:
Are the changes in this class still necessary?

Reply:
This is needed for the 0-1 scenario, where we originally don't have the config file in place. This scenario should contain some configURL, if not, we can't watch the parent directory for changes. Since the current solution (testClass.getClassLoader().getResource(resourceName)) returns null if the file does not exists, we need to pass the configURL somehow. This is the workaround for that. Maybe there's a better solution, but this is the solution I could think of that requires the least amount of work.